com.woltlab.wcf/templates/boxRecentActivitySidebar.tpl

```diff
@@ -1,6 +1,6 @@
 <ul class="sidebarItemList">
 	{foreach from=$eventList item=event}
-		<li class="box24{if $__wcf->getUserProfileHandler()->isIgnoredUser($event->getUserProfile()->userID)} ignoredUserContent{/if}">
+		<li class="box24{if $__wcf->getUserProfileHandler()->isIgnoredUser($event->getUserProfile()->userID, 2)} ignoredUserContent{/if}">
```

Comment:
If only class constants could be used in templates…

Reply:
I opted to preserve the magic '2', because it is the least ugly solution.